shub/schedule.py

```diff
@@ -0,0 +1,45 @@
+from ConfigParser import NoSectionError, NoOptionError
```

Comment:
When our dependencies add py3 support we might do the same for `shub`; so I think we should keep new code py3-compliant.